### ISSUE TITLE
No popup tooltip on right menu

### ISSUE THREAD

**Issue by AntMadeira** [openstreetmap/iD#8781]
### How to reproduce the issue?

Since last update, tooltips on the right menu are not showing. They had a brief description of the menu and a key shortcut, but that's no longer there when hovering over the icon.

### Which iD Editor versions do you see the issue on?

Released version at openstreetmap.org/edit

### Which browsers are you seeing this problem on?

Firefox

**Comment by mbrzakovic** [comment]
It seems that https://github.com/openstreetmap/iD/pull/8685 caused this regression.

More specifically this section:
https://github.com/openstreetmap/iD/blob/3e7fd6e0330a14319206c9c9eb47afb3d97ca8d9/css/80_app.css#L2967-L2968

**Comment by DujaOSM** [comment]
Was it also the cause of shortcut button failure? I cannot access the Map Features sidebar using the **F** key on 2.20.2. (Rather annoying since I use it frequently).

**Comment by AntMadeira** [comment]
> 
> 
> Was it also the cause of shortcut button failure? I cannot access the Map Features sidebar using the **F** key on 2.20.2. (Rather annoying since I use it frequently).

Yes, I noticed that also and I found out the new key is "U". The "F" is now used to the follow function.

**Comment by DujaOSM** [comment]
Thanks. I could get accustomed to the new shortcut key, if I could only see the tooltip to see which one it was. :)
